nestkernel/synaptic_element.cpp

```diff
@@ -120,7 +120,10 @@ nest::SynapticElement::set( const DictionaryDatum& d )
   double new_tau_vacant = tau_vacant_;
 
   // Store values
-  updateValue< double >( d, names::growth_rate, growth_rate_ );
+  if ( d->known( names::growth_rate ) )
+    growth_rate_ = getValue< double >( d, names::growth_rate );
```

Comment:
NEST guidelines ask for braces even around single line blocks (though I personally find it ugly, apparently it is for safety reasons), so I guess this ``if/else`` should span 4 more lines ;)